main.go

```diff
@@ -34,7 +35,50 @@ func initBucket() {
 	emptyVersionsFile, _ := json.Marshal(versions)
 	fmt.Println("Uploading empty " + versionFileKey + " file")
 
-	autoupdate.UploadS3File(s3Bucket, versionFileKey, bytes.NewReader(emptyVersionsFile))
+	err := autoupdate.UploadS3File(s3Bucket, versionFileKey, bytes.NewReader(emptyVersionsFile))
+	if err != nil {
+		panic(err)
+	}
+}
+func downloadLatest() {
+	appName := flag.Arg(1)
+	channel := flag.Arg(2)
+	s3Bucket := flag.Arg(3)
+	outputName := flag.Arg(4)
+	if flag.NArg() != 5 {
+		printUsage()
+		os.Exit(1)
+	}
+	versionFileKey := autoupdate.GetVersionFileKey(appName, channel)
+	fmt.Println("Fetching " + versionFileKey + "file")
```

Comment:
@MichalPodeszwa should we add a space like so to make it read better or is this the expected functionality?

Before
```go
fmt.Println("Fetching " + versionFileKey + "file")
```
After
```go
fmt.Println("Fetching " + versionFileKey + " file")
```